frontend/src/Participant.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { RtcStream } from "@bandwidth/webrtc-browser";
+
+export interface ParticipantProps {
+    rtcStream: RtcStream;
+}
+
+const Participant: React.FC<ParticipantProps> = (props) => {
+    return (
+        <div>
+            <div className="row border">
```

Comment:
I'm not a UI guru, but this looks halfway decent. It could definitely be improved on though.